### PR TITLE
chore(cas): Cleanup AnchorService

### PR DESCRIPTION
**Pull request by stbrody**
No behavior change here, just refactoring and adding a test

### CODE REVIEW

**Comment by stbrody** [comment]
This PR is built on top of Janko's anchor service cleanup PR: https://github.com/ceramicnetwork/ceramic-anchor-service/pull/190

File: src/controllers/request-controller.ts [hunk 1]
```diff
@@ -9,20 +9,20 @@ import cors from 'cors';
...
+import RequestRepository from '../repositories/request-repository';
 
 @singleton()
 @Controller('api/v0/requests')
 @ClassMiddleware([cors()])
 export default class RequestController {
 
   constructor(@inject('anchorService') private anchorService?: AnchorService,
-              @inject('requestService') private requestService?: RequestService, ) {
+              @inject('requestRepository') private requestRepository?: RequestRepository, ) {
```
Comment (simonovic86) [inline src/controllers/request-controller.ts:25]: I intentionally use `service`s from `controller`s layer. Services can add more logic on top of CRUD repositories. In this case that's not true, but that can change in the future (service layer caching,  etc.).

**Review by simonovic86 — APPROVED** [review]
nice cleanup! I added just a comment regarding `controller<->service<->repository` flow.

btw, I needed to merge the `feat/cas-cleanup` and redeploy it on our infra. Feel free to create a PR to develop directly

File: src/controllers/request-controller.ts (hunk 1 above)
Comment (stbrody) [reply src/controllers/request-controller.ts:25]: Yep, makes total sense, I just figured that's easy enough to add back if/when we need it.  Having the extra indirection layer now with no logic in it just seemed confusing to me.

**Pull request merged**